create_manual_sc_seg/generate_qc.sh

```diff
@@ -73,17 +80,16 @@ cd $PATH_DATA_PROCESSED/derivatives/labels/$SUBJECT/anat
 file_gt="${file}_lesion-manual"
 file_seg="${file}_seg-manual"
 
-# Make sure the GT metadata is a valid JSON object
-if [[ ! -s ${file_gt}.json ]]; then
-  echo "{}" >> ${file_gt}.json
-fi
-
 # Binarize the GTs because QC only accepts binary images
 sct_maths -i ${file_gt}.nii.gz -bin 0 -o ${file_gt}_bin.nii.gz
 
-# Run the QC
+# Generate sagittal lesion QC
+# Note: -s is the SC segmentation provided to crop the image
 sct_qc -i ${PATH_DATA_PROCESSED}/${SUBJECT}/anat/${file}.nii.gz -s ${file_seg}.nii.gz -d ${file_gt}_bin.nii.gz -p sct_deepseg_lesion -plane sagittal -qc ${PATH_QC} -qc-subject ${SUBJECT}
 
+# Generate single slice sagittal SC QC (to check FOV coverage (C/Th/L))
+sct_qc -i ${PATH_DATA_PROCESSED}/${SUBJECT}/anat/${file}.nii.gz -s ${file_seg}.nii.gz -p sct_label_vertebrae -qc ${PATH_QC} -qc-subject ${SUBJECT}
+
```

Comment:
This is interesting! So, not adding the `-d` flag gives a single slice QC instead of all sagittal slices?

Reply:
Notice that this line uses `-p sct_label_vertebrae` (more in [this comment](https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4011#issuecomment-1561736362)) while QC for lesions ([L88](https://github.com/ivadomed/model_seg_sci/blob/2b3e97d44741577c85e719e8a56e4adb1a3fc462/create_manual_sc_seg/generate_qc.sh#L88)) uses `-p sct_deepseg_lesion`.
I made it clear in https://github.com/ivadomed/model_seg_sci/pull/82/commits/cae3005759b53c756bcb6260be2152c50d1c5756

Reply:
ah that's smart! Yeah, I missed it completely, thanks for making it clear!